labs/lab02/skel/win/2-crc/crc.c

```diff
@@ -60,27 +65,45 @@ static DWORD GetSize(HANDLE file)
 	return dwSize;
 }
 
+static DWORD ReadChunk(CHAR *chunk, HANDLE *hFile)
```

Comment:
Why `HANDLE *hFile`? I think ` static DWORD ReadChunk(CHAR *chunk, HANDLE hFile)` would be better.